spec/erb_lint/cli_spec.rb

```diff
@@ -561,6 +561,17 @@ def run(processed_source)
             expect { subject }.to(output(/no files found\.\.\./).to_stderr)
             expect(subject).to(be(false))
           end
+
+          context "allowing for no matching files" do
+            let(:args) do
+              ["--config", config_file, "--enable-linter", "linter_with_errors,final_newline", "--allow-no-files", "--stdin", linted_file]
```

Comment:
This line is too long. Can you break it?

Reply:
Definitely! I put each option pair onto their own line.